rider-core/src/main/java/com/github/database/rider/core/api/dataset/SeedStrategy.java

```diff
@@ -28,7 +28,8 @@
     TRUNCATE_INSERT(new CompositeOperation(DatabaseOperation.TRUNCATE_TABLE, DatabaseOperation.INSERT)),
     INSERT(DatabaseOperation.INSERT),
     REFRESH(DatabaseOperation.REFRESH),
-    UPDATE(DatabaseOperation.UPDATE);
+    UPDATE(DatabaseOperation.UPDATE),
+    IDENTITY_INSERT(InsertIdentityOperation.CLEAN_INSERT);
```

Comment:
Can you update the javadoc with the new strategy?

Reply:
Done